### PR TITLE
Translated image_captioning from en to es

### PR DESCRIPTION
**Pull request by gisturiz**
# What does this PR do?

Translated image_captioning from en to es from issue #28936 began by @stevhliu. I will continue to go through the documentation and make the correct translations.

Fixes # (issue)
#28936

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#create-a-pull-request),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

@stevhliu 
Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: docs/source/es/tasks/image_captioning.md [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+})
+```
+
+El conjunto de datos tiene dos características, `image` y `text`.
+
+<Consejo>
+
+Muchos conjuntos de datos de subtitulación de imágenes contienen múltiples subtítulos por imagen. En esos casos, una estrategia común es muestrear aleatoriamente un subtítulo entre los disponibles durante el entrenamiento.
+
+</Consejo>
```
Comment (stevhliu) [inline docs/source/es/tasks/image_captioning.md:71]: You don't have to translate `<Tip>`, this is a part of the doc-builder's syntax :)

```suggestion
<Tip>

Muchos conjuntos de datos de subtitulación de imágenes contienen múltiples subtítulos por imagen. En esos casos, una estrategia común es muestrear aleatoriamente un subtítulo entre los disponibles durante el entrenamiento.

</Tip>
```

**Review by stevhliu — APPROVED** [review]
LGTM! I don't speak Spanish so I can't review the content. Maybe @aaronjimv would be interested in taking a look at your translation?

**Comment by aaronjimv** [comment]
Hi @stevhliu and @gisturiz, yes happy to help🤗. In general is very good, I would comment the following:

* Using `"Subtítulos de imagen"` instead of `"Subtitulación de Imágenes"` when translating `"Image captioning"`, as it sounds more natural.
* In the same way using `"el proceso de preprocesamiento"` instead of `"la canalización de preprocesamiento"` in the line 111.
* And translate this part `[this guide]` in line 153.

Remember to add this new documentation in the file `es/_toctree.yml` . You can read [this guide](https://github.com/huggingface/transformers/blob/main/docs/TRANSLATING.md) for more information.

**Comment by gisturiz** [comment]
Hi @aaronjimv thanks for your review, I've pushed the updates and should be good now. I'd appreciate any feedback. And thanks to you as well @stevhliu

**Comment by aaronjimv** [comment]
Hi @gisturiz, LGTM. 
My final comments would be remove the final `"g"` in `Subtítulos de imágenesg` on the `es/_toctree.yml` file. 
Change the times that `"la subtítulos de imágenes"` appears in the first paragraph to `"los subtítulos de imágenes"`.  
And remember to keep the tag `<Tip>` as original in English, since it is a part of the doc-builder's syntax.

**Comment by stevhliu** [comment]
Thanks again, love to see the collaboration here! 🔥 

I'll merge this once a fix for the failing test is added, so in the meantime there's nothing else you need to do 🙂

**Pull request closed without merging**